### DISCUSSION TITLE
More Mobs Variations

### DISCUSSION THREAD

**Discussion by Hyrule-Rose** [BedrockTweaks/Files#290] [Ideas]
So instead of just having  the normal regular pigs cows chickens bunnies Cats dogs/wolf, fox Etc There could be more variety skins for them?

**Comment by SquatchHunter** [comment]
its possible but idk if its "within scope" of BT to do it _for now_

this is very much something people do when designing an entire RP not just small tweaks to the otherwise Vanilla Experience, so its something we can look at down the track, but I am not sure _personally_ if it fits within this projects goal.

**Reply by AtlasOfEden** [reply]
To be fair, if you needed to add 10+ zombie skin alts, I'd 100% like to see the same done to the other hostile mobs. Namely Creepers and Endermen.

**Reply by SquatchHunter** [reply]
community packs will be done after addons are
we still dont have a timeline
